biomage/account/account.py

```diff
@@ -43,9 +45,7 @@ def create_account(full_name, email, userpool):
             --user-attributes \
                 "Name=email,Value={email}" \
                 "Name=name,Value='{full_name}'" \
-                "Name=email_verified,Value=true" """.format(
-            userpool=userpool, email=email, full_name=full_name
-        ),
```

Comment:
I deleted this because we're already using f-string so we don't need to use format anymore.